.gitlab-ci.yml

```diff
@@ -2,6 +2,7 @@ image: "$IMAGE"
 
 default:
   interruptible: true
+  timeout: 2h
```

Comment:
The rest of this file contains timeouts expressed in minutes (like 120m). But more importantly, you didn't put a larger timeout for fiat-crypto. Is that on purpose? And shouldn't we have a smaller default timeout and more explicitly mark all the jobs that take longer? We could use some of the stats that @JasonGross created this summer to get a precise idea of what is needed for what BTW (although this was with the shared runners).